src/ucp/api/ucp.h

```diff
@@ -1310,6 +1311,12 @@ typedef struct ucp_worker_params {
     * using @ref ucp_conn_request_query.
     */
     uint64_t                client_id;
+
+    /**
+     * Align this offset of the active message data by @ref am_alignment.
```

Comment:
"Align the active message data with an offset of this many bytes"?

Reply:
I agree that current definition is quite a bit confusing. Is not this is just an offset for payload to start ? 
How exactly UCT buffers get's align to this ? It seems that UCT alignment is the same but some padding is added. thanks